packages/pigment-css-react/tsconfig.json

```diff
@@ -7,12 +7,6 @@
     "composite": true,
     "noEmit": false,
     "resolveJsonModule": true,
-    "paths": {
-      "@mui/system": ["./packages/mui-system/src"],
-      "@mui/system/*": ["./packages/mui-system/src/*"],
-      "@mui/utils": ["./packages/mui-utils/src"],
```

Comment:
@brijeshb42 I noticed we don't list @mui/utils as direct dependency of @pigment-css/react, should we do it?

Reply:
We don't use it directly. Only through system. So it shouldn't be required.

Reply:
I saw an import here: https://github.com/mui/pigment-css/blob/master/packages/pigment-css-react/src/createUseThemeProps.js#L1, this is why I asked.

Reply:
Cool, I will add it.